tools/chromium_src/lit_mangler/lit_mangler_cli.ts

```diff
@@ -44,7 +44,7 @@ const runTypecheck = (genDir: string, files: string[]) => {
     const result = childProcess.spawnSync('tsc', ['-p', getTsConfigForFiles(genDir, files)])
     // Note: tsc in Windows doesn't return a status code on success (i.e. status === null).
     if (result.status ?? 0 !== 0) {
-        console.error('Typechecking failed:\n', result.stderr?.toString())
```

Comment:
On Windows, typechecking errors seem to be in `stderr` while on Linux they're on `stdout`. I'm adding both now to be safe